cpp_test/Main.cpp

```diff
@@ -0,0 +1,14 @@
+#include <opencv2/opencv.hpp>
+#include <iostream>
+#include "ImageCap.cpp"
+#include "ImageManipulation.cpp"
+
+
+int main(int argc, char *argv[])
```

Comment:
I'm seeing multiple mains here. If this is main, then the videoFeed.cpp should become a class that is runnable.

Reply:
The file videoFeed.cpp was just a test file that I used to see how to capture a video vs capture a single frame.